data/extension/init/templates/tinygo/envoy.access_loggers/default/main.go

```diff
@@ -28,7 +32,17 @@ func (l *accessLogger) OnPluginStart(configurationSize int) types.OnPluginStartS
 		proxywasm.LogCriticalf("failed to load config: %v", err)
 		return types.OnPluginStartStatusFailed
 	}
-	l.logMessage = string(data)
+
+	// Ignore comment lines starting with "#" in the configuration.
+	var lines []string
+	scanner := bufio.NewScanner(bytes.NewReader(data))
+	for scanner.Scan() {
+		line := scanner.Text()
+		if !strings.HasPrefix(line, "#") {
+			lines = append(lines, line)
+		}
+	}
```

Comment:
[nit] BTW, you could also move this parsing logic into SDK so that it could become a one-liner for extensions.

I think having this logic in SDK could also be helpful to extension examples over there.